.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    reviewers:
+      - "lzaoral"
```

Comment:
@vmihalko You should change the default reviewer to someone else when you get to this PR.